scripts/release

```diff
@@ -554,6 +570,34 @@ def do_build_appimage(ctx):
     print("success, files=", appimage_files)
 
 
+@cli.command()
+def docker_tags():
+    """
+    Prints the tags that should be used for any docker images that are built,
+    separated by whitespace.
```

Comment:
Ok, I've explained it more now. :-)

Reply:
Hm, I don't see it - `docker-tags` now includes `continuous` again, was this the intention? So in `release.sh` it will do a `docker tag libertem/libertem:continuous libertem/libertem:continuous`, correct? I'm fine with this, if it works, I'm just wondering...

Reply:
Yes, precisely. I wasn't 100 % happy with pushing an image before it went through QA. It broke the logical structure of the pipeline for me. The "push" step is not executed often anyway. So now the "continuous" image is official together with the other deploy. It also keeps that code path in continuous use. :-)